Source/inv.cpp

```diff
@@ -75,14 +75,14 @@ const InvXY InvRect[73] = {
 	{ 538, 336 }, // inv row 4
 	{ 567, 336 }, // inv row 4
 	{ 596, 336 }, // inv row 4

```

Comment:
To fix inventory display, above row should be  moved by WIDTH_DIFF only

so 
 {408, 305} // belt
became
 { 408 + WIDTH_DIFF, 305} // belt

Reply:
one at a time ;p main panel was first, now I've dealt with inventory / stuff already

Reply:
I was just a way to help ;)